docker/dev/config_docker.json

```diff
@@ -82,6 +82,14 @@
         "ShowEmailAddress": true,
         "ShowFullName": true
     },
+    "LegalAndSupportSettings": {
```

Comment:
This doesn't have to be a 1:1 mapping to the UI.  If it's just links then maybe LinkSettings (0/5)